ovirt_imageio/client/_tool.py

```diff
@@ -21,8 +22,11 @@ def main():
     _download.register(parser)
     args = parser.parse()
 
+    is_file = os.path.isfile(args.log_file)
+    if not is_file:
+        logging.StreamHandler(args.log_file)
```

Comment:
This line has no effect :-)

Reply:
True! It was the default stream value of the `basicConfig` what made it work. It could be better documented in the official docs.